### DISCUSSION TITLE
Please Post Your Tasks

### DISCUSSION THREAD

**Discussion by MikeCoraluzzi** [mlab-upenn/gokart-mechatronics#2] [General]
Hi everyone!

Please post the Tasks you're working on now and also the Tasks you expect to work on in the future.

This will enable us to make Issues and Project Board cards for them as needed and have them all visible in one place.

Thank you!
Mike

**Comment by Aurunima** [comment]
Aurchie's task:
Mimic a vehicle simulator container similar to the one here: https://github.com/aws-samples/demo-soafee-aws-iotfleetwise/tree/main/vsim
